code/game/objects/machinery/sentries.dm

```diff
@@ -1088,6 +1086,15 @@
 		visible_message("<span class='notice'>The [name] powers down and goes silent.</span>")
 		update_icon()
 
+/obj/machinery/marine_turret/premade/dumb/hostile
+	name = "malfunctioning UA 571-C sentry gun"
+	desc = "Oh god oh fuck."
+	turret_flags = TURRET_LOCKED|TURRET_ON|TURRET_BURSTFIRE|TURRET_IMMOBILE
+
+/obj/machinery/marine_turret/premade/dumb/hostile/attack_hand(mob/living/user)
+	to_chat(user,"<span class='warning'>\The [src.name] refuses to cooperate!</span>")
```

Comment:
```suggestion
	to_chat(user,"<span class='warning'>\The [src] refuses to cooperate!</span>")
```

Reply:
That is not how it works. Instead it will be The the sentry refuses to coperate. On all coderbases, it uses \The [src.name].